runtime/compiler/env/j9methodServer.cpp

```diff
@@ -1041,16 +1041,12 @@ TR_ResolvedJ9JITaaSServerMethod::isInlineable(TR::Compilation *comp)
       }
    }
 
-bool
-TR_ResolvedJ9JITaaSServerMethod::isWarmCallGraphTooBig(uint32_t bcIndex, TR::Compilation *comp)
-   {
-   _stream->write(JITaaS::J9ServerMessageType::ResolvedMethod_isWarmCallGraphTooBig, _remoteMirror, bcIndex);
-   return std::get<0>(_stream->read<bool>()); 
-   }
-
 void
 TR_ResolvedJ9JITaaSServerMethod::setWarmCallGraphTooBig(uint32_t bcIndex, TR::Compilation *comp)
    {
+   // set the variable in the server IProfiler
+   TR_ResolvedJ9Method::setWarmCallGraphTooBig(bcIndex, comp);
```

Comment:
By calling `TR_ResolvedJ9Method::setWarmCallGraphTooBig` the server will try to set the flag into its own **global** IProfiler which is the wrong thing. The server needs to access the per client cache )which has a different structure than the global IProfiler hashtable.

Reply:
`setWarmCallGraphTooBig` sets the flag in the bytecode entry returned by `profilingSample`.
I thought that calling `profilingSample` on the server already returns an entry from the per-client cache. Wouldn't this make calling `TR_ResolvedJ9Method::setWarmCallGraphTooBig` ok?

Reply:
You are right. I missed that `setWarmCallGraphTooBig` use the overridden `profilingSample` which knows how to access the information per client